backend/redirectioneaza/common/validators.py

```diff
@@ -0,0 +1,16 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+def url_validator(value):
+    if value in ("www", "admin", "api"):
```

Comment:
We should also prevent the "redirectioneaza" subdomain.
And some other service subdomains like "ftp", "dns", "ns1", "ns2", ...

Reply:
I only added the subdomains you suggested.
This is controllable from the admin and we'll check along the way if we should think about adding any new subdomains to the list.